R/semicolon_linter.R

```diff
@@ -6,6 +6,42 @@
 #'   semicolons (e.g. as in `x; y`, i.e., on the same line of code) are allowed.
 #' @param allow_trailing Logical, default `FALSE`. If `TRUE`, "trailing"
 #'   semicolons (i.e., those that terminate lines of code) are allowed.
+#'
+#' @examples
+#' # will produce lints
+#' lint(
+#'   text = "a <- 1;",
+#'   linters = semicolon_linter()
+#' )
+#'
+#' lint(
+#'   text = "a <- 1; b <- 1",
```

Comment:
WDYT about replacing this with an online function example, or adding a third example? that's the other place this is tempting to use in ad hoc code....